index.js

```diff
@@ -719,27 +721,21 @@ Command.prototype = {
         prevCommand.delegate_(command);
 
       // apply beforeInit options
-      command.setOptions(
```

Comment:
With this change, normalization for `--foo bar baz --foo quux` will run 2 times:
- `[foo, bar]`
- `[quux]`
For every passed option with same name

Without it only 1 time: `[foo, bar, quux]`